src/components/MultipleAvatars.tsx

```diff
@@ -279,7 +279,7 @@ function MultipleAvatars({
                         <View style={[secondAvatarStyles, secondAvatarStyle, icons[1].type === CONST.ICON_TYPE_WORKSPACE ? StyleUtils.getAvatarBorderRadius(size, icons[1].type) : {}]}>
                             {icons.length === 2 ? (
                                 <UserDetailsTooltip
-                                    accountID={icons[1].id}
+                                    accountID={Number(icons[1].id)}
```

Comment:
can't we just make the type `number | string` will there be any issue?

Reply:
If you look at onyx types (`src/types/onyx`), you can see that `accountID` is always a number, to stay consistent I typed it with `number`, wdyt?

Reply:
I think either way will work lets leave it.